### PR TITLE
added notes for 1.23.0 release

### PR DESCRIPTION
**Pull request by paynejacob**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,16 +1,22 @@
-## 1.22.3 (Unreleased)
+## 1.23.0 (March 31, 2022)
 
 FEATURES:
 
+* The Harvester node driver is now supported. https://github.com/rancher/terraform-provider-rancher2/pull/861
```
Comment (nunix) [inline CHANGELOG.md]:
```suggestion
* The Harvester node driver is now supported. See [#861](https://github.com/rancher/terraform-provider-rancher2/pull/861).
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,16 +1,22 @@
...
+* Added support for KEv2 cluster certificate rotation. https://github.com/rancher/terraform-provider-rancher2/pull/882
```
Comment (nunix) [inline CHANGELOG.md]:
```suggestion
* Added support for KEv2 cluster certificate rotation. See [#882](https://github.com/rancher/terraform-provider-rancher2/pull/882).
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,16 +1,22 @@
...
+* Added support for node pool health checks on KEv2 clusters. https://github.com/rancher/terraform-provider-rancher2/pull/889
```
Comment (nunix) [inline CHANGELOG.md]:
```suggestion
* Added support for node pool health checks on KEv2 clusters. See [#889](https://github.com/rancher/terraform-provider-rancher2/pull/889).
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,16 +1,22 @@
...
 
 
 ENHANCEMENTS:
 
+* `hetzner_node_driver` Now supports specifying `server_labels`. https://github.com/rancher/terraform-provider-rancher2/issues851
```
Comment (nunix) [inline CHANGELOG.md]:
```suggestion
* `hetzner_node_driver` now supports specifying `server_labels`. See [#851](https://github.com/rancher/terraform-provider-rancher2/pull/851).
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,16 +1,22 @@
...
+* `drain_before_delete` Can now be specified on `rancher2_cluster_v2` resources. https://github.com/rancher/terraform-provider-rancher2/issues890
```
Comment (nunix) [inline CHANGELOG.md]:
```suggestion
* `drain_before_delete` can now be specified on `rancher2_cluster_v2` resources. See [#890](https://github.com/rancher/terraform-provider-rancher2/pull/890).
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,16 +1,22 @@
...
 * Added support to import `rancher2_cloud_credential` resources
```
Comment (nunix) [inline CHANGELOG.md]:
```suggestion
* Added support for `rancher2_cloud_credential` imported resources.
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,16 +1,22 @@
...
 * Added support to `rancher2_node_template` resource for server labels in hetzner_config, https://github.com/rancher/terraform-provider-rancher2/issues/657
```
Comment (nunix) [inline CHANGELOG.md]:
```suggestion
* Added `rancher2_node_template` label for server resources in `hetzner_config`. See [#657](https://github.com/rancher/terraform-provider-rancher2/issues/657).
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,16 +1,22 @@
...
 
 BUG FIXES:
 
+* Fixed imported KEv2 clusters having improper changes on import. https://github.com/rancher/rancher/issues/36128
```
Comment (nunix) [inline CHANGELOG.md]:
```suggestion
* Importing KEv2 clusters with the Rancher client have their config correctly rewritten. See [#36128](https://github.com/rancher/rancher/issues/36128).
```

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,16 +1,22 @@
...
 * Fix race condition for empty kubeconfig generation for `rancher2_cluster_sync` resource, https://github.com/rancher/terraform-provider-rancher2/issues/849
```
Comment (nunix) [inline CHANGELOG.md]:
```suggestion
* Fix race condition for empty kubeconfig generation for `rancher2_cluster_sync` resource. See [#849](https://github.com/rancher/terraform-provider-rancher2/issues/849).
```

**Review by nunix — COMMENTED** [review]
Changed the ending of the issues to be aligned with the Rancher release notes and rephrased + corrected links.

**Review by nunix — APPROVED** [review]

**Review by snasovich — APPROVED** [review]

**Pull request merged**